File: 66-plus-one/plus-one.java
class Solution {
    public int[] plusOne(int[] digits) {
        int n=digits.length-1;
        for(int i=n;i>=0;i--){
            if(digits[i]<9){
                digits[i]=digits[i]+1;
                return digits;
            }
            digits[i]=0;
                
        }
        int[] ne=new int[n+2];
        ne[0]=1;
        return ne;
          
        
    }
}

File: 66-plus-one/plus-one.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        num = 0
        for i in range(len(digits)):
            num = num*10 + digits[i]
        
        num = num+1

        l = []
        while num > 0:
            l.append(num % 10)
            num//=10
        return l[::-1]


            
